src/xinterpreter.cpp

```diff
@@ -92,6 +92,9 @@ namespace xpyt
 
         // Add get_ipython to global namespace
         py::globals()["get_ipython"] = get_kernel_module().attr("get_ipython");
+
+        m_has_ipython = get_kernel_module().attr("has_ipython").cast<bool>();
```

Comment:
Where do you set the `has_ipython` value in the kernel module?

Reply:
[Here](https://github.com/jupyter-xeus/xeus-python/pull/289/files#diff-d1e8fed419dfbdd955a412b7b0111e28R389) or [here](https://github.com/jupyter-xeus/xeus-python/pull/289/files#diff-d1e8fed419dfbdd955a412b7b0111e28R384)

Reply:
Oh yeah sorry I missed it